driver/src/main/java/oracle/nosql/driver/iam/InstancePrincipalsProvider.java

```diff
@@ -129,7 +129,7 @@ public static class InstancePrincipalsProviderBuilder {
             "169.254.169.254";
 
         /* The default value for HTTP request timeouts in milliseconds */
-        private static final int DEFAULT_TIMEOUT_MS = 120_000;
+        private static final int DEFAULT_TIMEOUT_MS = 10_000;
```

Comment:
10s still seems too long for this default. Most users won't override it and with the default operation timeout at 5s, it seems that 5s or less is appropriate